### DISCUSSION TITLE
Procedure to propose a pull request and merge

### DISCUSSION THREAD

**Discussion by sangwan-smriti** [camicroscope/caMicroscope#777] [Q&A]
hello everyone, hope you are having a great day, 
i  had some doubts so all help would be welcome:)

In issues section where each issue had a "asignee" field which was blank. 
1. What is procedure to propose an issue and pull request i.e in which section do i have to write a report? 
3. Can i work on any bug/issue i like or ill have to get it approved by our mentor @birm. 
4. Can i work on solving an issue which has been proposed by someone earlier?

i want to start working on dark mode and accessiblity of website, what steps will i have to follow to ensure im im following all guidelines and procedures of community?

the original website and  cloned one  has a different ui on local host, is it supposed to be like this? Or im doing something wrong

**Comment by birm** [comment]
You shouldn't need to assign issues or pull requests, we're just pretty far behind now, slowly catching up!
As for the other questions:
1. Pull requests should be clear (what is being done, how, and why); you shouldn't need to write a lot here. Just be clear!
2. We'll look at all Pull Requests (as we get to them at least!) No need to get permission.
3. Yes, you can work on issues proposed by someone else, we don't promote claiming/locking people out of things if we can help it.

What do you mean about the original website? camicroscope.org? The UI is different there since it's mostly holding info, and just happens to point to a demo too.

Hope this helps!

**Reply by sangwan-smriti** [reply]
It was helpful, thanks.
And yes i meant camicroscope.org by original website, its different from index.html of cloned repo opened on local host. is it supposed to be like that? and the changes are to be made there?
